docs/downloads/automation-library/standard/pr-description-checklist/pr_description_checklist.cm

```diff
@@ -0,0 +1,21 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+automations:
+  pr_description_checklist:
+    if:
+      - {{ pr.description | includes(regex=r/\-\[\]/) }}
```

Comment:
the original regex doesnt cover GitHub markdown checkbox `- [ ]` which has spaces
```suggestion
      - {{ pr.description | includes(regex=r/\- \[ \]/) }}
```